### PR TITLE
Add custom_js site param for enabling custom js

### PR DESCRIPTION
**Pull request by mrozlukasz**
I've hit that when adding Cookie Consent on my web page. https://github.com/insites/cookieconsent

### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Allows to add custom javascript source files in html head page. I need this for features like Cookie Consent. 

### Issues Resolved

Haven't seen any related

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [ ] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [ ] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [ ] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -60,9 +60,13 @@
     {{ end }}
 
     {{ range .Site.Params.custom_css }}
-      <link rel="stylesheet" href="{{ . | relURL }}">
+      <link rel="stylesheet" href="{{ . | relURL }}"/>
```
Comment (luizdepra) [inline layouts/_default/baseof.html]:
```suggestion
      <link rel="stylesheet" href="{{ . | relURL }}" />
```
Just adding a space here.

**Comment by luizdepra** [comment]
Good. I need to ask one more thing.
Add this new config into [exampleSite/config.toml](https://github.com/luizdepra/hugo-coder/blob/master/exampleSite/config.toml) just bellow `custom_css`.

**Comment by mrozlukasz** [comment]
All set. Thanks for the response and review.

**Comment by luizdepra** [comment]
Thank you!

**Comment by mrozlukasz** [comment]
No problem :-)

**Pull request merged**